airflow/models/dagcode.py

```diff
@@ -128,9 +122,9 @@ def bulk_sync_to_db(cls, filelocs, session=None):
                 os.path.getmtime(correct_maybe_zipped(fileloc)), tz=timezone.utc)
 
             if (file_modified - timedelta(seconds=120)) > old_version.last_updated:
-                orm_dag_code = DagCode(fileloc)
+                orm_dag_code = DagCode(fileloc, cls._get_code_from_db(fileloc))
```

Comment:
Is it necessary? This causes an n + 1 problem. Can you add tests for the number of queries for this method?

Reply:
Fixed, I will add number of queries count when I PR against Master.